src/components/PreviewComment/index.js

```diff
@@ -110,18 +110,18 @@ export const styles = StyleSheet.create({
   lineBeforeProfile: {
     height: 9,
     borderLeftWidth: 1,
-    borderLeftColor: COLORS.gray1,
+    borderLeftColor: '#C4C4C4',
     marginLeft: 8
   },
   container: (totalComment) => ({
     borderLeftWidth: 1,
     marginHorizontal: SIZES.base,
-    borderLeftColor: totalComment >= 1 ? COLORS.gray1 : COLORS.white
+    borderLeftColor: totalComment >= 1 ? '#C4C4C4' : '#fff'
   }),
   username: {
     fontFamily: fonts.inter[700],
     fontSize: normalizeFontSize(10),
-    color: COLORS.blackgrey,
+    color: '#828282',
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The hardcoded color '#828282' should be replaced with a constant for better maintainability.
```diff
- color: '#828282',
+ color: colors.grey,
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->